src/SparseCSCInterface/SparseCSCInterface.jl

```diff
@@ -1,5 +1,5 @@
 module SparseCSCInterface
-using SparseArrays, LinearAlgebra
+import SparseArrays, LinearAlgebra
```

Comment:
Is an import necessary? Are we adding methods?

Reply:
I looked at this the other way around:
`import` is safer than an unqualified `using`, with the only difference being that it avoids importing many unknown and unused methods into the namespace with possibility of name clashes etc?

In this case, the only effect of this change was to enforce `SparseArrays.nnz` which looks like what was intended, and caught a couple of places where an unqualified `nnz` was inadvertently used which then was potentially confusing (well confused me anyway!).

Many thanks for fantastic work making this package available btw!

Reply:
> In this case, the only effect of this change was to enforce `SparseArrays.nnz`

Good point.